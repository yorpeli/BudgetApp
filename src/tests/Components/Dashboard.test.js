import React from 'react';
import { shallow } from 'enzyme';
import DashboardPage from '../../components/Dashboard';

test ('Render Dashboard', ()=>{
    const wrapper = shallow(<DashboardPage/>);
    expect(wrapper).toMatchSnapshot();
});