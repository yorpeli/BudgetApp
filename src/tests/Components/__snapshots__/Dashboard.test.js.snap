// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Dashboard 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
